### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by MalikIdreesHasanKhan**
Typos in the code have been fixed.

### CODE REVIEW

**Comment by Megamouse** [comment]
Please remove the replied file

File: rpcs3/util/serialization.hpp [hunk 1]
```diff
@@ -52,7 +52,7 @@ namespace utils
 		std::vector<u8> data;
 		usz pos = umax;
 
-		// Checks if this strcuture is currently used for serialization
+		// Checks if this structure is currently used for serialization
```
Comment (elad335) [inline rpcs3/util/serialization.hpp:55]: Change to "instance" while you're at it.

**Comment by Scr34mik** [comment]
While using this build, wipeout HD Fury's save data is not being read. I had to rollback to 13155

**Comment by MSuih** [comment]
It is not related to these chnges. See #11370

**Pull request merged**